content/release-10-18-0/announcements-10-18-0.md

```diff
@@ -91,6 +91,18 @@ As announced with [release 10.17](/release-10-17-0/announcements-10-17-0), to im
 This change prevents that an attacker could have access to all users, in case the administrator account was compromised.
 Note that the administrator will still have the option to force the user to reset the password on the next login or disable the user.
 
+### Microservice infrastructure changes
+
+#### Planned
+
+##### Breaking change: Use of Linux cgroup v2
+As of release 2024, microservices must use a Linux cgroup v2 aware application runtime. When executing microservices which are not compatible with cgroup v2 on Cumulocity IoT in version 2024 and higher it might happen that the information provided by the application runtime concerning available CPU and memory is not correct. This might lead to incorrect memory and thread allocation in the microservice container process. 
```

Comment:
Is the name of the next release already fix? Otherwise we should perhaps better phrase it differently, for example: As of release 2024 > in a future release
in version 2024 and higher > in a release higher than 10.18

Reply:
@BeateRixen I just took over the changes which you have proposed on the 10.17 PR. The text for announcing this change in 10.17 and 10.18 should be the same I think.